### PR TITLE
Half Worm volume and mass changes

### PR DESCRIPTION
**Pull request by Karol1223**
#### Summary
Balance "Number changes for half worms"

#### Purpose of change
On the devcord it was lately brought up that it's better for a giant worm to split if you want to butcher it as two half worms give more resources than one giant worm - the laws of physics have been foiled again.

#### Describe the solution
Half worms had their numbers changed to roughly half of giant worm's (the mass is exactly half, the volume is rounded down) while the large worms now use the numbers half worms used to have (large worms always used them, anyway, as a part of copy_from)

### CODE REVIEW

**Comment by Maleclypse** [comment]
I've set it to rerun all tests to see if that causes json styling test to run

**Comment by Karol1223** [comment]
It doesn't appear to have changed anything... Weird, the only test lest is Windows Build but it doesn't add the specific label...

File: data/json/monsters/insect_spider.json [hunk 1]
```diff
@@ -74,8 +74,8 @@
     "default_faction": "worm",
     "bodytype": "snake",
     "species": [ "WORM" ],
-    "volume": "62500 ml",
-    "weight": "81500 g",
+    "volume": "45 L",
+    "weight": "60 kg",
```
Comment (Fris0uman) [inline data/json/monsters/insect_spider.json:78]: But this is still more than half the full worm, no?

Comment (Karol1223) [reply data/json/monsters/insect_spider.json:78]: It is more than half the full _smallest_ worm, yes. That size variant does not split. When it grows it becomes 120kg and 92.5 l of volume and only then can it split.

**Pull request merged**